warehousing-alternatives-compare-table.md

```diff
@@ -0,0 +1,18 @@
+## Differences between Cloudant to Db2 data flow solutions
+
+### Source options from Cloudant:
+
+technology | available document extract endpoints | notes
+--- | --- | --- 
+Apache Bahir Spark connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
+IBM Cloud Functions | `/{db}/_changes`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | - 
```

Comment:
There should be a note here that a Cloud Function trigger is only available for _changes

Reply:
Added in c35a74c.